test/e2e/performanceprofile/functests/7_performance_kubelet_node/cgroups.go

```diff
@@ -218,7 +202,7 @@ var s_ = Describe("[performance] Cgroups and affinity", Ordered, func() {
 			testutils.CustomBeforeAll(func() {
 				initialProfile = profile.DeepCopy()
 			})
-			It("[test_id:64100] matches with ovs process affinity", func() {
+			It("[test_id:64100] matches with ovs process affinity", Label("fun1"), func() {
```

Comment:
`Label("fun1")` I would use more descriptive name

Reply:
Removed the label, it was used only for testing.